jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/Validator.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.cli.buildfile;
+
+import com.google.common.base.Preconditions;
+import java.util.Collection;
+
+/**
+ * Utility helper class to detect errors in parsed yaml values. This class is mostly concerned with
+ * error message formatting, checking is delegated to guava.
+ */
+public class Validator {
+
+  /**
+   * Checks if an object reference is not null.
+   *
+   * @param value the object in question
+   * @param propertyName the equivalent 'yaml' property name
+   * @throws NullPointerException if {@code value} is null
+   */
+  public static void checkNotNull(Object value, String propertyName) {
+    Preconditions.checkNotNull(value, "Property '" + propertyName + "' cannot be null");
```

Comment:
Should we annotate `@Nullable` on `value`?

BTW, Guava `checkNotNull` returns the input object like the following, although we don't need to follow the same pattern, of course.

```java
this.something = Preconditions.checkNotNull(something);
```

Reply:
Yeah I thought about that, it doesn't seem necessary here.